### DISCUSSION TITLE
BUG? when using powergrid

### DISCUSSION THREAD

**Discussion by blockhunts** [wireui/wireui#590] [Q&A] [answered]
In my page, I have a datetime picker and a select.
It works as intended before, but glitched when I include powergrid styles and scripts. 

```
@powerGridStyles
@powerGridScripts
```
*It works fine again, when I removed the two line above.

Here's the screenshot 
![report](https://user-images.githubusercontent.com/38546608/209596776-3bd2f36b-24a9-4e21-8bcd-be16cbb109f5.gif)

I know that powergrid is not part of WireUI component, but perhaps someone know why this thing happen in the first place 😃

**Comment by luanfreitasdev** [comment, marked as answer]
Hello!, can you check if alpine cdn is included in the `config/livewire-powergrid.php` ? I believe that's it 
This configuration already came out in version 4

**Reply by blockhunts** [reply]
Hello, thanks for the reply.
I reinstalled powergrid just now, and publish the config.
I didn't changed anything, and the powergrid version is 3.7

![image](https://user-images.githubusercontent.com/38546608/214734623-3aa51b27-2b39-4827-8899-a3c228fcb22a.png)

**Reply by luanfreitasdev** [reply]
I believe you already have Alpine installed, if so you can leave alpine_cnd as null

**Reply by blockhunts** [reply]
Yes it does!
It works now after I changed the config to null.

Now I understand what the doc means here 😅😂
Sorry for the trouble, because I failed to understand the doc

![image](https://user-images.githubusercontent.com/38546608/214735931-d85db159-46fc-4900-b582-5ce95248b903.png)
